protobuf_definitions/message_formats.proto

```diff
@@ -899,4 +901,15 @@ message Imu {
   Vector3 gyroscope = 2; // Angular velocity (rad/s)
   Vector3 magnetometer = 3; // Magnetic field (μT)
   float temperature = 4; // Temperature (°C)
+}
+
+/**
+  * Gamma ray sensor spectrometer data
+  */
+message SpectrometerData {
```

Comment:
Would these properties be relevant for any spectrometer, or only for the medusa? Thinking about the generic name of the message - to avoid naming issues in future (not that we will implement that many spectrometers - but just something to think through).

Reply:
Good point, made the name more specific